apps/condo/domains/user/schema/RegisterNewUserService.js

```diff
@@ -156,10 +156,16 @@ const RegisterNewUserService = new GQLCustomSchema('RegisterNewUserService', {
                 }
                 const sendChannels = [{
                     to: { phone: userData.phone },
+                    type: REGISTER_NEW_USER_MESSAGE_TYPE,
                 }]
                 if (!isEmpty(userData.email)) {
                     sendChannels.push({
                         to: { email: userData.email },
+                        type: REGISTER_NEW_USER_MESSAGE_TYPE,
+                    })
+                    sendChannels.push({
```

Comment:
It looks strange. Why you want to send two emails for user?

Reply:
Because we need to send an email with username and password and an email with a greetings